contracts/CourtRoom.sol

```diff
@@ -7,24 +7,32 @@ import "./ensabstract.sol";
 
 contract CaseContract is Owned {
 
+  struct promise{
+		address beneficiary;
+		uint256 blockNumber;
+		uint8 sig_v;
+		bytes32 sig_r;
+		bytes32 sig_s;
+	}
 	struct claim {
 		bytes32 claimId;
 		address plaintiff;
 		bytes32[] evidence;
+		promise   servicePromise;
 		uint status;
 		bool valid;
```

Comment:
If there is not enough evident so the claim is rejected .the client can submit a new one